Source/Core/Core/HW/WiiSave.cpp

```diff
@@ -89,52 +92,54 @@ void CWiiSaveCrypted::ExportAllSaves()
   u32 success = 0;
   for (const u64& title : titles)
   {
-    CWiiSaveCrypted export_save{"", title};
-    if (export_save.m_valid)
+    WiiSave export_save{title};
+    if (export_save.Export())
       success++;
   }
   SuccessAlertT("Successfully exported %u save(s) to %s", success,
                 (File::GetUserPath(D_USER_IDX) + "private/wii/title/").c_str());
 }
 
-CWiiSaveCrypted::CWiiSaveCrypted(const std::string& filename, u64 title_id)
-    : m_encrypted_save_path(filename), m_title_id(title_id)
+WiiSave::WiiSave()
 {
   memcpy(m_sd_iv, "\x21\x67\x12\xE6\xAA\x1F\x68\x9F\x95\xC5\xA2\x23\x24\xDC\x6A\x98", 0x10);
```

Comment:
Would moving this from a `memcpy`/string to an array still count as "minor cleanup"? ;)